### DISCUSSION TITLE
carbon

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-native#392] [Ideas]
I heard that carbon could be a successor for C++. Do you think it could be interesting to convert MapLibre GL Native to carbon?

https://github.com/carbon-language/carbon-lang

**Comment by ntadej** [comment]
I would not jump into it too fast. The main concern is the cross-platform compiler support and how long this project will even live.

**Reply by wipfli** [reply]
Ah yeah google projects tend to come and go...
